redis/deprecated/redisbp/monitored_client.go

```diff
@@ -30,12 +37,40 @@ func NewMonitoredFailoverClient(name string, opt *redis.FailoverOptions) *redis.
 	return client
 }
 
+// ClusterClient extends redis cluster client with a functional Wait function
+type ClusterClient struct {
+	*redis.ClusterClient
+}
+
+// Wait blocks the current client until all the previous write commands are
+// successfully transferred and acknowledged by at least the specified number of replicas.
+func (cc *ClusterClient) Wait(ctx context.Context, key string, numSlaves int, timeout time.Duration) (replicas int64, err error) {
```

Comment:
🔕 I guess this is tied to redis command so very unlikely to change in the future, but I'd still prefer to convert the args into a struct so we can avoid breaking changes if we need to add/remove args. e.g.

```
Wait(ctx context.Context, args WaitArgs) (replicas int64, err error)
```

Reply:
Ok sure